examples/complete-http/outputs.tf

```diff
@@ -135,3 +135,8 @@ output "api_endpoint" {
   description = "FQDN of an API endpoint"
   value       = "https://${aws_route53_record.api.fqdn}"
 }
+
+output "authorizer_id" {
```

Comment:
Please copy the outputs you added in the module as-is. Like it is on lines 123-126.

Reply:
No need to add extra output `authorizer_id` here

Reply:
Thanks @antonbabenko for the review - copied the output directly into the example as requested!